docs/guide/acquiring_data/fido.rst

```diff
@@ -180,131 +167,346 @@ client for results, in this case VSO.::
+    2012-03-04 00:00:00.000 2012-03-04 23:59:59.999       LYRA ...      ESA     3
     <BLANKLINE>
     1 Results from the RHESSIClient:
-         Start Time           End Time      Instrument ... Source Provider
-    ------------------- ------------------- ---------- ... ------ --------
-    2012-03-04 00:00:00 2012-03-04 23:59:59     RHESSI ... RHESSI     NASA
+           Start Time               End Time        Instrument ... Source Provider
+    ----------------------- ----------------------- ---------- ... ------ --------
+    2012-03-04 00:00:00.000 2012-03-04 23:59:59.999     RHESSI ... RHESSI     NASA
     <BLANKLINE>
     3 Results from the VSOClient:
-           Start Time               End Time        ... Extent Type      Info
-                                                    ...
-    ----------------------- ----------------------- ... ----------- --------------
-    2012-03-03 22:57:40.000 2012-03-04 00:33:20.000 ... PARTIAL_SUN RHESSI level-0
-    2012-03-04 00:33:20.000 2012-03-04 01:45:40.000 ... PARTIAL_SUN RHESSI level-0
-    2012-03-04 01:45:40.000 2012-03-04 02:09:00.000 ... PARTIAL_SUN RHESSI level-0
+           Start Time               End Time        ...   Size        Info
+                                                    ...  Mibyte
+    ----------------------- ----------------------- ... -------- --------------
+    2012-03-03 22:57:40.000 2012-03-04 00:33:20.000 ... -0.00098 RHESSI level-0
+    2012-03-04 00:33:20.000 2012-03-04 01:45:40.000 ... -0.00098 RHESSI level-0
+    2012-03-04 01:45:40.000 2012-03-04 02:09:00.000 ... -0.00098 RHESSI level-0
     <BLANKLINE>
     <BLANKLINE>
 
-Indexing search results
-***********************
 
-The `~sunpy.net.fido_factory.UnifiedResponse` that Fido returns can be
-indexed to access a subset of the search results. When doing this, the
-results should be treated as a two-dimensional array in which the first
-dimension corresponds to the clients which have returned results and the
-second to the records returned.
+Working with Search Results
+***************************
 
-For example, the following code returns a response containing LYRA data from
-the `~sunpy.net.dataretriever.LYRAClient`, and EVE data from the
-`~sunpy.net.vso.VSOClient`::
+:meth:`Fido.search <sunpy.net.fido_factory.UnifiedDownloaderFactory.search` can make multiple queries to multiple clients in one search.
+This means that the results of a call to search can contain many sets of records, called responses, from many clients.
+The results of a search are represented in a `~sunpy.net.fido_factory.UnifiedResponse` object, which provides access to all the response tables and allows some operations to be performed on all the results at once.
+`~sunpy.net.fido_factory.UnifiedResponse` acts both like a two dimensional array, where the first dimension is the response index and the second index is the row index, and a dictionary where you can index the responses by the name of the client.
```

Comment:
I actually hadn't realized you could index the response as a dictionary - is there a simple way to do `res.keys()` or something to figure out what the names of the clients are (I mean I know the names are somewhat clear but for example, it indexes as `XRS` instead of `XRSClient`)

Reply:
The algorithm is everything before `Client` lower cased. I have added a `keys()` method which gives you the list of possible options. Do you think there is a need to make it more obvious than this as to what responses they correspond to?